reconcile/openshift_serviceaccount_tokens.py

```diff
@@ -35,6 +36,31 @@ def construct_sa_token_oc_resource(name: str, sa_token: str) -> OR:
     )
 
 
+def service_account_token_request(name: str) -> OR:
+    """Create a service account token secret for a given service account."""
+    body = {
+        "apiVersion": "v1",
+        "kind": "Secret",
+        "type": "kubernetes.io/service-account-token",
+        "metadata": {
+            # service-account-name-<random-number>
+            "name": f"{name}-token-{random.randrange(99999):05d}",
+            "annotations": {
+                "kubernetes.io/service-account.name": name,
+            },
+        },
+    }
+    return OR(
+        body,
+        # We are marking this token secret as "unmanaged" because we just want to create it
+        # and not manage it in the future.
+        # Openshift will delete this token secret automatically if the service account is deleted.
+        f"{QONTRACT_INTEGRATION}-unmanaged",
+        QONTRACT_INTEGRATION_VERSION,
+        error_details=name,
```

Comment:
any case we want to delete the token but keep the service account?

Reply:
Not I can think of. 🤷

Reply:
if we don't mark it as `unmanaged`, just treat it like other resources in ri, will it conflict with token auto deletion when service account deleted?